OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -47,7 +47,7 @@ public static Actor ChooseResupplier(Actor self, bool unreservedOnly)
 					&& a.Owner == self.Owner
 					&& rearmInfo.RearmActors.Contains(a.Info.Name)
 					&& (!unreservedOnly || Reservable.IsAvailableFor(a, self)))
-				.ClosestTo(self);
+				.ClosestToWithPathFrom(self);
```

Comment:
this activity is aircraft exclusive so it doesn't make sense to pathfind

Reply:
`ClosestToWithPathFrom` has knowledge that aircraft can path anywhere. So I think I'll keep this as-is so the special casing of Aircraft remains contained within the helpers.